src/shared.ts

```diff
@@ -21,6 +22,7 @@ export const componentNameMapping: Record<
   ConnectElementHTMLName
 > = {
   "account-onboarding": "stripe-connect-account-onboarding",
+  documents: "stripe-connect-documents",
```

Comment:
Can we add them in the same way/order as https://github.com/stripe/connect-js/pull/99/files? That will make a future merge of these branches easier.